homeTask_5/homeTask_5/Book.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace homeTask_5
+{
+    class Book
+    {
+        private String name="";
+        private int pageCount;
+        private String uniqueId = "";
+        public Book(String name, int pageCount, String uniqueId)
+        {
+            this.name = name;
+            this.pageCount = pageCount;
+            this.uniqueId = uniqueId;
+        }
+        public Book()
```

Comment:
use auto properties instead of create methods for each property.

Reply:
Done. @EgorAntonovich Please review this task.